pkg/cloud/metadata/lingjun.go

```diff
@@ -0,0 +1,51 @@
+package metadata
+
+import (
+	"encoding/json"
+)
+
+var LingjunConfigFile = "/host/etc/eflo_config/lingjun_config"
+
+type LingjunMetaData struct {
+	RegionId          string   `json:"RegionId"`
+	ZoneId            string   `json:"ZoneId"`
+	NodeId            string   `json:"NodeId"`
+	InstanceType      string   `json:"InstanceType"`
+	EniMac            string   `json:"EniMac"`
+	AckNicName        string   `json:"AckNicName"`
+	NimitzNetworkMode string   `json:"NimitzNetworkMode"`
+	NimitzInterface   []string `json:"NimitzInterface"`
+}
+
+var LingjunConfig = map[MetadataKey]string{
+	RegionID:     "RegionId",
+	ZoneID:       "ZoneId",
+	InstanceID:   "NodeId",
+	InstanceType: "InstanceType",
+}
+
+func NewLingJunMetadata(data []byte) (*LingjunMetaData, error) {
+	lm := LingjunMetaData{}
+	err := json.Unmarshal(data, &lm)
+	if err != nil {
+		return nil, err
+	}
+	return &lm, nil
+}
+
+func (m *LingjunMetaData) Get(key MetadataKey) (string, error) {
+
+	if v, ok := LingjunConfig[key]; ok {
+		switch v {
+		case "RegionId":
+			return m.RegionId, nil
+		case "ZoneId":
+			return m.ZoneId, nil
+		case "NodeId":
+			return m.NodeId, nil
+		case "InstanceType":
+			return m.InstanceType, nil
+		}
+	}
+	return "", ErrUnknownMetadataKey
```

Comment:
```suggestion
		switch v {
		case "RegionId":
			return m.RegionId, nil
		case "ZoneId":
			return m.ZoneId, nil
		case "NodeId":
			return m.NodeId, nil
		case "InstanceType":
			return m.InstanceType, nil
		default:
			return "", ErrUnknownMetadataKey
		}
```
And remove `LingjunConfig` completely.